drivers/bridge/bridge.go

```diff
@@ -667,16 +707,16 @@ func (d *driver) createNetwork(config *networkConfiguration) (err error) {
 
 	// Add inter-network communication rules.
 	setupNetworkIsolationRules := func(config *networkConfiguration, i *bridgeInterface) error {
-		if err := network.isolateNetwork(networkList, true); err != nil {
-			if err = network.isolateNetwork(networkList, false); err != nil {
+		if err := network.isolateNetwork(iptables.IPv4, networkList, true); err != nil {
```

Comment:
curious why we are setting this to ipv4 ?

Reply:
I think this was done to not change the previous behavior of the lib.
Basically the IPv6 patch is not 100% complete.

Sadly I am (at least for now) not really familiar with the code base of libnetwork or docker at all so I can not real say if it is relevant to add here also IPv6 handling.

Reply:
I think we should be passing the IPtables version in this API, this call will finally call `setINC` which should hopefully fo the right right

Reply:
moved handling of IP version inside `network.isolateNetwork` and also handle IPv6